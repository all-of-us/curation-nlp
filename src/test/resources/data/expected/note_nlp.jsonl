{"note_nlp_id":0,"note_id":3002,"section_concept_id":0,"snippet":"test_snippet","offset":"0-100","lexical_variant":"test_variant","note_nlp_concept_id":100,"note_nlp_source_concept_id":100,"nlp_system":"CLAMP 1.7.2","nlp_date":"2022-11-10","nlp_datetime":"2022-11-10 10:10:10","term_exists":"False","term_temporal":"1 year","term_modifiers":"TermModifier"}
{"note_nlp_id":0,"note_id":1001,"section_concept_id":0,"snippet":"test_snippet","offset":"0-100","lexical_variant":"test_variant","note_nlp_concept_id":100,"note_nlp_source_concept_id":100,"nlp_system":"CLAMP 1.7.2","nlp_date":"2022-11-10","nlp_datetime":"2022-11-10 10:10:10","term_exists":"False","term_temporal":"1 year","term_modifiers":"TermModifier"}
