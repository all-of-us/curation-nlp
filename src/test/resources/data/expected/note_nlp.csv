note_nlp_id,note_id,section_concept_id,snippet,offset,lexical_variant,note_nlp_concept_id,note_nlp_source_concept_id,nlp_system,nlp_date,nlp_datetime,term_exists,term_temporal,term_modifiers
0,3004,0,test_snippet,0-100,test_variant,100,100,CLAMP 1.7.2,2022-11-10,2022-11-10 10:10:10,False,1 year,TermModifier
0,1001,0,test_snippet,0-100,test_variant,100,100,CLAMP 1.7.2,2022-11-10,2022-11-10 10:10:10,False,1 year,TermModifier
0,3002,0,test_snippet,0-100,test_variant,100,100,CLAMP 1.7.2,2022-11-10,2022-11-10 10:10:10,False,1 year,TermModifier